train.py

```diff
@@ -143,10 +151,15 @@ def main():
         train(model, x_train, y_train, params["lr"], params["weight_decay"])
         torch.save(model.state_dict(), "model.pt")
         # Evaluate and checkpoint.
-        metrics, actual, predicted = evaluate(model, x_test, y_test)
-        for k, v in metrics.items():
-            live.log_metric(k, v)
+        metrics_train, _, _ = evaluate(model, x_train, y_train)
+        for k, v in metrics_train.items():
+            live.log_metric(f"{k}_train", v)
```

Comment:
I would recommend:

```suggestion
            live.log_metric(f"train/{k}", v)
```

IMO, it look nicer in the tree view and the table

Reply:
Made this change, but there were a couple reasons I used the other syntax initially:
1. `train/{k}` probably breaks on Windows.
2. I think it's not that clear whether the header in `train/acc.tsv` should be `acc` or `train/acc` (right now, it's `train/acc`).